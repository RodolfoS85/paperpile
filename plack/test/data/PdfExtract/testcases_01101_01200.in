---
file: ../../../PDFREPOSITORY/001101.pdf
---
file: ../../../PDFREPOSITORY/001102.pdf
---
file: ../../../PDFREPOSITORY/001103.pdf
---
file: ../../../PDFREPOSITORY/001104.pdf
---
file: ../../../PDFREPOSITORY/001105.pdf
---
file: ../../../PDFREPOSITORY/001106.pdf
---
file: ../../../PDFREPOSITORY/001107.pdf
---
file: ../../../PDFREPOSITORY/001108.pdf
---
file: ../../../PDFREPOSITORY/001109.pdf
---
file: ../../../PDFREPOSITORY/001110.pdf
---
file: ../../../PDFREPOSITORY/001111.pdf
---
file: ../../../PDFREPOSITORY/001112.pdf
---
file: ../../../PDFREPOSITORY/001113.pdf
---
file: ../../../PDFREPOSITORY/001114.pdf
---
file: ../../../PDFREPOSITORY/001115.pdf
---
file: ../../../PDFREPOSITORY/001116.pdf
---
file: ../../../PDFREPOSITORY/001117.pdf
---
file: ../../../PDFREPOSITORY/001118.pdf
---
file: ../../../PDFREPOSITORY/001119.pdf
---
file: ../../../PDFREPOSITORY/001120.pdf
---
file: ../../../PDFREPOSITORY/001121.pdf
---
file: ../../../PDFREPOSITORY/001122.pdf
---
file: ../../../PDFREPOSITORY/001123.pdf
---
file: ../../../PDFREPOSITORY/001124.pdf
---
file: ../../../PDFREPOSITORY/001125.pdf
---
file: ../../../PDFREPOSITORY/001126.pdf
---
file: ../../../PDFREPOSITORY/001127.pdf
---
file: ../../../PDFREPOSITORY/001128.pdf
---
file: ../../../PDFREPOSITORY/001129.pdf
---
file: ../../../PDFREPOSITORY/001130.pdf
---
file: ../../../PDFREPOSITORY/001131.pdf
---
file: ../../../PDFREPOSITORY/001132.pdf
---
file: ../../../PDFREPOSITORY/001133.pdf
---
file: ../../../PDFREPOSITORY/001134.pdf
---
file: ../../../PDFREPOSITORY/001135.pdf
---
file: ../../../PDFREPOSITORY/001136.pdf
---
file: ../../../PDFREPOSITORY/001137.pdf
---
file: ../../../PDFREPOSITORY/001138.pdf
---
file: ../../../PDFREPOSITORY/001139.pdf
---
file: ../../../PDFREPOSITORY/001140.pdf
---
file: ../../../PDFREPOSITORY/001141.pdf
---
file: ../../../PDFREPOSITORY/001142.pdf
---
file: ../../../PDFREPOSITORY/001143.pdf
---
file: ../../../PDFREPOSITORY/001144.pdf
---
file: ../../../PDFREPOSITORY/001145.pdf
---
file: ../../../PDFREPOSITORY/001146.pdf
---
file: ../../../PDFREPOSITORY/001147.pdf
---
file: ../../../PDFREPOSITORY/001148.pdf
---
file: ../../../PDFREPOSITORY/001149.pdf
---
file: ../../../PDFREPOSITORY/001150.pdf
---
file: ../../../PDFREPOSITORY/001151.pdf
---
file: ../../../PDFREPOSITORY/001152.pdf
---
file: ../../../PDFREPOSITORY/001153.pdf
---
file: ../../../PDFREPOSITORY/001154.pdf
---
file: ../../../PDFREPOSITORY/001155.pdf
---
file: ../../../PDFREPOSITORY/001156.pdf
---
file: ../../../PDFREPOSITORY/001157.pdf
---
file: ../../../PDFREPOSITORY/001158.pdf
---
file: ../../../PDFREPOSITORY/001159.pdf
---
file: ../../../PDFREPOSITORY/001160.pdf
---
file: ../../../PDFREPOSITORY/001161.pdf
---
file: ../../../PDFREPOSITORY/001162.pdf
---
file: ../../../PDFREPOSITORY/001163.pdf
---
file: ../../../PDFREPOSITORY/001164.pdf
---
file: ../../../PDFREPOSITORY/001165.pdf
---
file: ../../../PDFREPOSITORY/001166.pdf
---
file: ../../../PDFREPOSITORY/001167.pdf
---
file: ../../../PDFREPOSITORY/001168.pdf
---
file: ../../../PDFREPOSITORY/001169.pdf
---
file: ../../../PDFREPOSITORY/001170.pdf
---
file: ../../../PDFREPOSITORY/001171.pdf
---
file: ../../../PDFREPOSITORY/001172.pdf
---
file: ../../../PDFREPOSITORY/001173.pdf
---
file: ../../../PDFREPOSITORY/001174.pdf
---
file: ../../../PDFREPOSITORY/001175.pdf
---
file: ../../../PDFREPOSITORY/001176.pdf
---
file: ../../../PDFREPOSITORY/001177.pdf
---
file: ../../../PDFREPOSITORY/001178.pdf
---
file: ../../../PDFREPOSITORY/001179.pdf
---
file: ../../../PDFREPOSITORY/001180.pdf
---
file: ../../../PDFREPOSITORY/001181.pdf
---
file: ../../../PDFREPOSITORY/001182.pdf
---
file: ../../../PDFREPOSITORY/001183.pdf
---
file: ../../../PDFREPOSITORY/001184.pdf
---
file: ../../../PDFREPOSITORY/001185.pdf
---
file: ../../../PDFREPOSITORY/001186.pdf
---
file: ../../../PDFREPOSITORY/001187.pdf
---
file: ../../../PDFREPOSITORY/001188.pdf
---
file: ../../../PDFREPOSITORY/001189.pdf
---
file: ../../../PDFREPOSITORY/001190.pdf
---
file: ../../../PDFREPOSITORY/001191.pdf
---
file: ../../../PDFREPOSITORY/001192.pdf
---
file: ../../../PDFREPOSITORY/001193.pdf
---
file: ../../../PDFREPOSITORY/001194.pdf
---
file: ../../../PDFREPOSITORY/001195.pdf
---
file: ../../../PDFREPOSITORY/001196.pdf
---
file: ../../../PDFREPOSITORY/001197.pdf
---
file: ../../../PDFREPOSITORY/001198.pdf
---
file: ../../../PDFREPOSITORY/001199.pdf
---
file: ../../../PDFREPOSITORY/001200.pdf
