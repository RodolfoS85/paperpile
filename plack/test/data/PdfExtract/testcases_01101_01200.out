---
authors: 'Quackenbush, John'
title: COMPUTATIONAL ANALYSIS OF MICROARRAY DATA
---
authors: 'Rabani, Michal and Kertesz, Michael and Segal, Eran'
doi: 10.1073/pnas.0803169105
title: Computational prediction of RNA structural motifs involved in posttranscriptional regulatory processes
---
authors: 'Raes, Jeroen and Van de Peer, Yves'
doi: 10.1016/j.tig.2005.05.013
title: Functional divergence of proteins through frameshift mutations
---
authors: 'Ragan, Chikako and Cloonan, Nicole and Grimmond, Sean M. and Zuker, Michael and Ragan, Mark A.'
doi: 10.1371/journal.pone.0005745
title: Transcriptome-Wide Prediction of miRNA Targets in Human and Mouse Using FASTH
---
authors: 'Rajewsky, Nikolaus'
doi: 10.1038/ng1798
title: microRNA target predictions in animals
---
authors: 'Rajewsky, Nikolaus and Vergassola, Massimo and Gaul, Ulrike and Siggia, Eric D'
title: Computational detection of genomic cis-regulatory modules applied to body patterning in the early Drosophila embryo
---
authors: 'Ramakrishnan, V.'
title: HISTONE STRUCTURE AND THE ORGANIZATION OF THE NUCLEOSOME
---
authors: 'Ramaswamy, Sridhar and Tamayo, Pablo and Rifkin, Ryan and Mukherjee, Sayan and Yeang, Chen-Hsiang and Angelo, Michael and Ladd, Christine and Reich, Michael and Latulippe, Eva and Mesirov, Jill P. and Poggio, Tomaso and Gerald, William and Loda, Massimo and Lander, Eric S. and Golub, Todd R.'
doi: 10.1073/pnas.211566398
title: Multiclass cancer diagnosis using tumor gene expression signatures
---
authors: 'Rao, Christopher V. and Wolf, Denise M. and Arkin, Adam P.'
title: 'Control, exploitation and tolerance of intracellular noise'
---
authors: 'Rapold, Roland F. and Mattice, Wayne L.'
title: Introduction of Short and Long Range Energies To Simulate Real Chains on the 2nnd Lattice
---
authors: 'Rasmussen, Anders Aamann and Johansen, Jesper and Nielsen, Jesper S. and Overgaard, Martin and Kallipolitis, Birgitte and Valentin-Hansen, Poul'
doi: 10.1111/j.1365-2958.2009.06688.x
title: A conserved small RNA promotes silencing of the outer membrane protein YbfM
---
authors: 'Reddy, D. Ashok and Mitra, Chanchal K.'
title: Comparative Analysis of Transcription Start Sites Using Mutual Information
---
authors: 'Reed, Floyd A. and Aquadro, Charles F.'
doi: 10.1016/j.tig.2006.07.005
title: 'Mutation, selection and the future of human evolution'
---
authors: 'Reeder, Jens and Giegerich, Robert'
doi: 10.1186/1471-2105-5-104
title: 'Design, implementation and evaluation of a practical pseudoknot folding algorithm based on thermodynamics'
---
authors: 'Reese, Martin G. and Hartzell, George and Harris, Nomi L. and Ohler, Uwe and Abril, Josep F. and Lewis, Suzanna E.'
title: Genome Annotation Assessment in Drosophila melanogaster
---
authors: 'Reeves, Raymond'
title: 'Molecular biology of HMGA proteins: hubs of nuclear function'
---
authors: 'Reidys, Christian M. and Stadler, Peter F.'
title: COMBINATORIAL LANDSCAPES
---
authors: 'Remenyi, Attila and Tomilin, Alexey and Scholer, Hans R. and Wilmanns, Matthias'
title: Differential activity by DNA-induced quarternary structures of POU transcription factors
---
authors: 'Ren, Jihong and Rastegari, Baharak and Condon, Anne and Hoos, Holger H.'
doi: 10.1261/rna.7284905
title: 'HotKnots: Heuristic prediction of RNA secondary structures including pseudoknots'
---
authors: 'Ren, Bing and Robert, Francois and Wyrick, John J. and Aparicio, Oscar and Jennings, Ezra G. and Simon, Itamar and Zeitlinger, Julia and Schreiber, Jorg and Hannett, Nancy and Kanin, Elenita and Volkert, Thomas L. and Wilson, Christopher J.'
title: Genome-Wide Location and Function of DNA Binding Proteins
---
authors: 'Resch, Alissa and Xing, Yi and Modrek, Barmak and Gorlick, Michael and Riley, Robert and Lee, Christopher'
doi: 10.1021/pr034064v
title: Assessing the Impact of Alternative Splicing on Domain Interactions in the Human Proteome
---
authors: 'Reynolds, Angela and Leake, Devin and Boese, Queta and Scaringe, Stephen and Marshall, William S and Khvorova, Anastasia'
doi: 10.1038/nbt936
title: Rational siRNA design for RNA interference
---
authors: 'Rhoades, Matthew W. and Reinhart, Brenda J. and Lim, Lee P. and Burge, Christopher B. and Bartel, Bonnie and Bartel, David P.'
title: Prediction of Plant MicroRNA Targets
---
authors: 'Richmond, Timothy J. and Davey, Curt A.'
title: The structure of DNA in the nucleosome core
---
authors: 'Richter, Sara and Cao, Hong and Rana, Tariq M.'
doi: 10.1021/bi0159579
title: Specific HIV-1 TAR RNA Loop Sequence and Functional Groups Are Required for Human Cyclin T1-Tat-TAR Ternary Complex Formation
---
authors: 'Richter, Sara and Ping, Yueh-Hsin and Rana, Tariq M.'
doi: 10.1073/pnas.122119999
title: 'TAR RNA loop: A scaffold for the assembly of a regulatory switch in HIV replication'
---
authors: 'Riechmann, Jose Luis and Ito, Toshiro and Meyerowitz, Elliot M.'
title: Non-AUG Initiation of AGAMOUS mRNA Translation in Arabidopsis thaliana
---
authors: 'Rife, Jason P and Moore, Peter B'
title: The structure of a methylated tetraloop in 16S ribosomal RNA
---
authors: 'Rife, Jason P. and Stallings, Sarah C. and Correll, Carl C. and Dallas, Anne and Steitz, Thomas A. and Moore, Peter B.'
title: Comparison of the Crystal and Solution Structures of Two RNA Oligonucleotides
---
authors: 'Rigoutsos, Isidore and Floratos, Aris'
title: 'Combinatorial pattern discovery in biological sequences: the TEIRESIAS algorithm'
---
authors: 'Marinescu, Voichita D and Kohane, Isaac S and Riva, Alberto'
doi: 10.1186/1471-2105-6-79
title: 'MAPPER: a search engine for the computational identification of putative transcription factor binding sites in multiple genomes'
---
authors: 'Rivas, Elena and Eddy, Sean R.'
title: 'The language of RNA: a formal grammar that includes pseudoknots'
---
authors: 'Rivas, Elena and Eddy, Sean R.'
title: A Dynamic Programming Algorithm for RNA Structure Prediction Including Pseudoknots
---
authors: 'Rivas, Elena and Eddy, Sean R'
title: Noncoding RNA gene detection using comparative sequence analysis
---
authors: 'Rivas, Elena and Klein, Robert J. and Jones, Thomas A. and Eddy, Sean R.'
title: Computational identification of noncoding RNAs in E. coli by comparative genomics
---
authors: 'Rivera, Maria C. and Lake, James A.'
title: The ring of life provides evidence for a genome fusion origin of eukaryotes
---
authors: 'Robert, Jason Scott'
title: 'Interpreting the homeobox: metaphors of gene action and activation in development and evolution'
---
authors: 'Roberts, Gavin C. and Gooding, Clare and Mak, Ho Yi and Proudfoot, Nicholas J. and Smith, Christopher W. J.'
title: Co-transcriptional commitment to alternative splice site selection
---
authors: 'Roberts, Gavin C and Smith, Christopher WJ'
title: 'Alternative splicing: combinatorial output from the genome'
---
authors: 'Robins, Harlan and Li, Ying and Padgett, Richard W.'
doi: 10.1073/pnas.0500775102
title: Incorporating structure to predict microRNA targets
---
authors: 'Robinson, G. E. and Ben-Shahar, Y.'
title: 'Social behavior and comparative genomics: new genes or new gene regulation?'
---
authors: 'Robison, Keith and McGuire, Abigail Manson and Church, George M.'
title: A Comprehensive Library of DNA-binding Site Matrices for 55 Proteins Applied to the Complete Escherichia coli K-12 Genome
---
authors: 'Robson-Dixon, Nicole D. and Garcia-Blanco, Mariano A.'
doi: 10.1074/jbc.M312747200
title: MAZ Elements Alter Transcription Elongation and Silencing of the Fibroblast Growth Factor Receptor 2 Exon IIIb
---
authors: 'RÃ˜DLAND, Einar Andreas'
title: 'Pseudoknots in RNA Secondary Structures: Representation, Enumeration, and Prevalence'
---
authors: 'Rogan, Peter K. and Faux, Brian M. and Schneider, Thomas D.'
title: Information Analysis of Human Splice Site Mutations
---
authors: 'Romero, Pedro R. and Zaidi, Saima and Fang, Ya Yin and Uversky, Vladimir N. and Radivojac, Predrag and Oldfield, Christopher J. and Cortese, Marc S. and Sickmeier, Megan and Le Gall, Tanguy and Obradovic, Zoran and Dunker, A. Keith'
doi: 10.1073/pnas.0507916103
title: Alternative splicing in concert with protein intrinsic disorder enables increased functional diversity in multicellular organisms
---
authors: 'Roshan, Usman and Chikkagoudar, Satish and Livesay, Dennis R'
doi: 10.1186/1471-2105-9-61
title: Searching for evolutionary distant RNA homologs within genomic sequences using partition function posterior probabilities
---
authors: 'Roshan, Usman and Livesay, Dennis R.'
title: 'Probalign: Multiple sequence alignment using partition function posterior probabilities'
---
authors: 'Ross, Eric D. and Keating, Anne M. and James Maher III, L.'
doi: 10.1006/jmbi.2000.3562
title: DNA Constraints on Transcription Activation in Vitro
---
authors: 'Rost, Burkhard and Fariselli, Piero and Casadio, Rita'
title: Topology prediction for helical transmembrane proteins at 86% accuracy
---
authors: 'Rother, M. and Wilting, R. and Commans, S. and Bock, A.'
doi: 10.1006/jmbi.2000.3756
title: Identification and Characterisation of the Selenocysteine-specific Translation Factor SelB from the Archaeon Methanococcus jannaschii
---
authors: 'Roubelakis, Maria G and Zotos, Pantelis and Papachristoudis, Georgios and Michalopoulos, Ioannis and Pappa, Kalliopi I and Anagnou, Nicholas P and Kossida, Sophia'
doi: 10.1186/1471-2105-10-S6-S20
title: 'Human microRNA target analysis and gene ontology clustering by GOmir, a novel stand-alone application'
---
authors: 'Roulet, Emmanuelle and Bucher, Philippe and Schneider, Ralf and Wingender, Edgar and Dusserre, Yves and Werner, Thomas and Mermod, Nicolas'
doi: 10.1006/jmbi.2000.3614
title: Experimental Analysis and Computer Prediction of CTF/NFI Transcription Factor DNA Binding Sites
---
authors: 'Rouzina, Ioulia and Bloomfield, Victor A.'
title: 'DNA Bending by Small, Mobile Multivalent Cations'
---
authors: 'VAN ROY, Peter'
title: 'Concepts, Techniques, and Models of Computer Programming'
---
authors: 'Rudnick, Stephen I. and Swaminathan, Jyothishmathi and Sumaroka, Marina and Liebhaber, Stephen and Gewirtz, Alan M.'
doi: 10.1073/pnas.0805781105
title: Effects of local mRNA structure on posttranscriptional gene silencing
---
authors: 'Ruiz-Echevarria, Maria J. and Gonzalez, Carlos I. and Peltz, Stuart W.'
title: 'Identifying the right stop: determining how the surveillance complex recognizes and degrades an aberrant mRNA'
---
authors: 'Ruml, Wheeler'
title: Incomplete Tree Search using Adaptive Probing
---
authors: 'RUNDLÃ–F, ANNA-KLARA and ARNÃ‰R, Elias S. J.'
title: 'Regulation of the Mammalian Selenoprotein Thioredoxin Reductase 1 in Relation to Cellular Phenotype, Growth, and Signaling Events'
---
authors: 'Russell, Anthony G. and Charette, J. Michael and Spencer, David F. and Gray, Michael W.'
doi: 10.1038/nature05228
title: An early evolutionary origin for the minor spliceosome
---
authors: 'Saakian, David B. and Hu, Chin-Kun'
doi: 10.1103/PhysRevE.69.046121
title: Solvable biological evolution model with a parallel mutation-selection scheme
---
authors: 'Sabatti, Chiara and Rohlin, Lars and Lange, Kenneth and Liao, James C.'
title: 'Vocabulon: a dictionary model approach for reconstruction and localization of transcription factor binding sites'
---
authors: 'Sachenbacher, Martin and Williams, Brian C.'
title: On-Demand Bound Computation for Best-First Constraint Optimization
---
authors: 'Saeys, Yvan and Degroeve, Sven and Aeyels, Dirk and RouzÃ©, Pierre and Van de Peer, Yves'
title: 'Feature selection for splice site prediction: A new method using EDA-based feature ranking'
---
authors: 'Sakabe, Noboru Jo and Vibranovski, Maria Dulcetti and JosÃ© de Souza, Sandro'
title: A bioinformatics analysis of alternative exon usage in human genes coding for extracellular matrix proteins
---
authors: 'Sakakibara, Yasubumi and Asai, Kiyoshi and Sato, Kengo'
title: Stem Kernels for RNA Sequence Analyses
---
authors: 'Salari, Raheleh and Backofen, Rolf and Sahinalp, S. Cenk'
title: Fast prediction of RNA-RNA Interaction
---
authors: 'Salehi-Ashtiani, Kourosh and Szostak, Jack W.'
title: In vitro evolution suggests multiple origins for the hammerhead ribozyme
---
authors: 'Sali, Andrej and Glaeser, Robert and Earnest, Thomas and Baumeister, Wolfgang'
title: From words to literature in structural proteomics
---
authors: 'Salzberg, Steven L. and Delcher, Arthur L. and Kasif, Simon and White, Owen'
title: Microbial gene identification using interpolated Markov models
---
authors: 'Salzberg, Steven'
title: Locating Protein Coding Regions in Human DNA using a Decision Tree Algorithm
---
authors: 'Samudrala, Ram and Levitt, Michael'
title: A comprehensive analysis of 40 blind protein structure predictions
---
authors: 'Samudrala, Ram and Xia, Yu and Huang, Enoch and Levitt, Michael'
title: Ab Initio Protein Structure Prediction Using a Combined Hierarchical Approach
---
authors: 'Samudrala, Ram and Huang, Enoch S. and Koehl, Patrice and Levitt, Michael'
title: Constructing side chains on near-native main chains for ab initio protein structure prediction
---
authors: 'Sandalova, Tatyana and Zhong, Liangwei and Lindqvist, Ylva and Holmgren, Arne and Schneider, Gunter'
doi: 10.1073/pnas.171178698
title: 'Three-dimensional structure of a mammalian thioredoxin reductase: Implications for mechanism and evolution of a selenocysteine-dependent enzyme'
---
authors: 'Sankoff, David'
title: Genome rearrangement with gene families
---
authors: 'Savaresi, Sergio M. and Boley, -Daniel L. and Gazzaniga, -Sergio Bittanti -Giovanna'
title: Choosing the cluster to split in bisecting divisive clustering algorithms
---
authors: 'Sazani, Peter and Kole, Ryszard'
doi: 10.1172/JCI200319547
title: Therapeutic potential of antisense oligonucleotides as modulators of alternative splicing
---
authors: 'Scamborova, Petra and Wong, Anthony and Steitz, Joan A.'
doi: 10.1128/MCB.24.5.1855-1869.2004
title: An Intronic Enhancer Regulates Splicing of the Twintron of Drosophila melanogaster prospero Pre-mRNA by Two Different Spliceosomes
---
authors: 'Schaal, Thomas D. and Maniatis, Tom'
title: Multiple Distinct Splicing Enhancers in the Protein-Coding Sequences of a Constitutively Spliced Pre-mRNA
---
authors: 'Schattner, Peter and BARBERAN-SOLER, Sergio and Lowe, Todd M.'
doi: 10.1261/rna.2210406
title: A computational screen for mammalian pseudouridylation guide H/ACA RNAs
---
authors: 'Scherr, Michaela and Rossi, John J. and Sczakiel, Georg and Patzel, Volker'
title: 'RNA accessibility prediction: a theoretical approach is consistent with experimental studies in cell extracts'
---
authors: 'Schindler, Stefanie and Szafranski, Karol and Hiller, Michael and Ali, Gul Shad and Palusa, Saiprasad G and Backofen, Rolf and Platzer, Matthias and Reddy, Anireddy SN'
doi: 10.1186/1471-2164-9-159
title: Alternative splicing at NAGNAG acceptors in Arabidopsis thaliana SR and SR-related protein-coding genes
---
authors: 'Schmidt, Jeanette P.'
title: ALL HIGHEST SCORING PATHS IN WEIGHTED GRID GRAPHS AND THEIR APPLICATION TO FINDING ALL APPROXIMATE REPEATS IN STRINGS
---
authors: 'Schmidt, Heiko A. and Strimmer, Korbinian and Vingron, Martin and von Haeseler, Arndt'
title: 'TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing'
---
authors: 'Schnall-Levin, Michael and Chindelevitch, Leonid and Berger, Bonnie'
title: 'Inverting the Viterbi Algorithm: An Abstract Framework for Structure Design'
---
authors: 'Schn, J Christian and Sibani, Paolo'
title: Properties of the energy landscape of network models for covalent glasses
---
authors: 'Schon, Astrid and Fingerhut, Christiane and Hess, Wolfgang R.'
doi: 10.1099/ijs.0.01983-0
title: Conserved and variable domains within divergent RNase P RNA gene sequences of Prochlorococcus strains
---
authors: 'Schonbrun, Jack and Dill, Ken A.'
doi: 10.1073/pnas.1735417100
title: Fast protein folding kinetics
---
authors: 'Schroeder, RenÃ©e and Barta, Andrea and Semrad, Katharina'
doi: 10.1038/nrm1497
title: STRATEGIES FOR RNA FOLDING AND ASSEMBLY
---
authors: 'Schulte, Christian'
title: Comparing Trailing and Copying for Constraint Programming
---
authors: 'Schultes, Erik A. and Bartel, David P.'
title: 'One Sequence, Two Ribozymes: Implications for the Emergence of New Ribozyme Folds'
---
authors: 'Schulz, Georg E.'
title: The structure of bacterial outer membrane proteins
---
authors: 'Schuster, Peter and Stadler, Peter F.'
title: Discrete Models of Biopolymers
---
authors: 'Schuster, Peter'
doi: 10.1007/s00706-008-0892-6
title: Modeling in biological chemistry. From biochemical kinetics to systems biology
---
authors: 'Schuster, Peter and Stadler, Peter F.'
title: Networks in Molecular Evolution
---
authors: 'Schuster, Stefan and Dandekar, Thomas and Fell, David'
title: Description of the algorithm for computing elementary flux modes
---
authors: 'Schuster, Stefan and Fell, David A. and Dandekar, Thomas'
title: A general definition of metabolic pathways useful for systematic organization and analysis of complex metabolic networks
---
authors: 'Schuster, Peter and Fontana, Walter and Stadler, Peter F. and Hofacker, Ivo L.'
title: 'From Sequences to Shapes and back: A Case Study in RNA Secondary Structures'
---
authors: 'Searls, David B.'
title: The language of genes
